
From: <janitor@sternwelten.at>

Kill KERNEL_VERSION duplicate.  Funny that it insn't even used in here

Signed-off-by: Domen Puncer <domen@coderock.org>
Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/media/video/videocodec.c |    5 -----
 1 files changed, 5 deletions(-)

diff -puN drivers/media/video/videocodec.c~kill-kernel_version-duplicate-in-videocodecc drivers/media/video/videocodec.c
--- 25/drivers/media/video/videocodec.c~kill-kernel_version-duplicate-in-videocodecc	Thu Sep 23 15:51:12 2004
+++ 25-akpm/drivers/media/video/videocodec.c	Thu Sep 23 15:51:12 2004
@@ -43,11 +43,6 @@
 #include <asm/uaccess.h>
 #endif
 
-#include <linux/version.h>
-#ifndef KERNEL_VERSION
-#define KERNEL_VERSION(a,b,c) ((a)*65536+(b)*256+(c))
-#endif
-
 #include "videocodec.h"
 
 static int debug = 0;
_
